<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFeature</class>
 <widget class="QDialog" name="CreateFeature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_group">
       <property name="text">
        <string>group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_group">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_count">
       <property name="text">
        <string>count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_count"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_common">
     <property name="text">
      <string>is common</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_actualNominal">
     <property name="text">
      <string>is nominal</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_nominalSystem">
       <property name="text">
        <string>nominal system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_nominalSystem"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_startSystem">
       <property name="text">
        <string>start system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_startSystem"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_destinationSystem">
       <property name="text">
        <string>destination system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_destinationSystem"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
     <item>
      <widget class="QToolButton" name="toolButton_mConfig">
       <property name="text">
        <string>measurement config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_create">
       <property name="text">
        <string>create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_cancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
